examples/cluster-workflow-template/clustertemplates.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ClusterWorkflowTemplate
+metadata:
+  name: cluster-workflow-template-whalesay-template
+spec:
+  templates:
+  - name: whalesay-template
+    inputs:
+      parameters:
+      - name: message
+    container:
+      image: docker/whalesay
+      command: [cowsay]
+      args: ["{{inputs.parameters.message}}"]
+---
+apiVersion: argoproj.io/v1alpha1
+kind: ClusterWorkflowTemplate
+metadata:
+  name: cluster-workflow-template-nested-template
+spec:
+  templates:
+  - name: whalesay-inner-template
+    templateRef:
```

Comment:
`tempalte.tempalteRef` is deprecated and should not be supported. Please see https://github.com/argoproj/argo/pull/2447

Reply:
Please remove/rework this example.

Reply:
sure